dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/JdbcEnrollmentAnalyticsManager.java

```diff
@@ -342,6 +357,119 @@ protected String getSelectClause( EventQueryParams params )
         return "select " + StringUtils.join( selectCols, "," ) + " ";
     }
 
+    /**
+     * Returns an encoded column name respecting the geometry/coordinate format.
+     * The given QueryItem must be of type COORDINATE.
+     *
+     * @param item the {@link QueryItem}
+     * @return the column selector or EMPTY if the item valueType is not
+     *         COORDINATE. ie.: ( select '[' ||
```

Comment:
Maybe shorten all this in the return Javadoc, a bit too much to include the entire SQL query.

Reply:
Done